sdk/identity/identity/review/identity.api.md

```diff
@@ -56,6 +56,18 @@ export interface AuthenticationRequiredErrorOptions {
     scopes: string[];
 }
 
+// @public
+export interface AuthorityValidationOptions {
```

Comment:
@KarishmaGhiya - has these been discussed / approved with all Azure SDKs? 

Afaik, Azure SDK will not expose "knownAuthorities" configuration, only a flag (true / false) which disabled instance metadata. Internally, this flag can just set knownAuthorities to the value of the authority, but I don't think the concept should be exposed as public API.

@rayluo can confirm this from MSAL side